Jenkinsfile

```
@@ -146,7 +152,7 @@ pipeline {
         sh 'nix-store --delete /nix/store/*docker-image*'
         withCredentials([file(credentialsId: 'kubeconfig', variable: 'KUBECONFIG')]) {
           sh 'kubectl get nodes -o wide'
-          sh "nix-shell --run './scripts/e2e-test.sh ${env.REGISTRY}'"
+          sh "nix-shell --run './scripts/e2e-test.sh -d /dev/nvme1n1 -t ${env.GIT_COMMIT_SHORT} -r ${env.REGISTRY}'"
```

```suggestion
          sh "nix-shell --run './scripts/e2e-test.sh --device /dev/nvme1n1 --tag \"${env.GIT_COMMIT_SHORT}\" --registry \"${env.REGISTRY}\"'"
```

Makes it easier for future readers. Not sure about quoting, do you think I did it right?